dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Accelerate module for PyOpenGL"
+HOMEPAGE="http://pyopengl.sourceforge.net/ https://pypi.org/project/PyOpenGL-accelerate/"
+MY_PN="PyOpenGL-accelerate"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+RDEPEND="dev-python/pyopengl[${PYTHON_USEDEP}]"
```

Comment:
As the CI check let you know, there is no python3.8 available for pyopengl yet, so this package can not work with python3.8 either.

Reply:
https://github.com/gentoo/gentoo/pull/14912 - dev-python/pyopengl version bump with py38